.github/workflows/tests-nightly.yml

```diff
@@ -308,12 +310,56 @@ jobs:
     runs-on: ubuntu-latest
     needs: [get-environment-variables, run-tests-on-release, add-review-and-merge-patch]
     steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        
       - name: Notify Slack
         env:
           JOB_DEPLOYMENT_KIND: staging
           JOB_STATUS: ${{ job.status }}
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_RELEASES_WEBHOOK_URL }}
           JOB_TITLE: "Test release workflow - ${{github.event.client_payload.project}} ${{github.event.client_payload.version}}"
           JOB_KIND: "release tests"
+        run: |
+          python3 .github/workflows/notify/notify-slack.py
+
+  send-slack-notification-scheduled-and-manually:
+    if: ${{always() && github.event_name != 'repository_dispatch'}}
+    defaults:
+      run:
+       shell: bash
+       working-directory: .github/workflows
+    runs-on: ubuntu-latest
+    needs: [run-tests-in-parallel]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+
+      - name: Install dependencies
+        run: |
+              npm ci
+
+      - name: Get tests results
+        id: get-tests-results
+        env: 
+         tests_status: ${{ needs.run-tests-in-parallel.outputs.status }}
+         dashboard_url: ${{ needs.run-tests-in-parallel.outputs.dashboard_url }}
+        run: |
+            node cypressTestsHelpers/getFailedTests.js \
+            --dashboard_url $dashboard_url 
+
+      - name: Notify Slack
+        if: steps.get-tests-results.outputs.testStatus == 'true'
+        env:
+          JOB_DEPLOYMENT_KIND: ${{ needs.run-tests-in-parallel.outputs.environment }}
+          JOB_STATUS: ${{ job.status }}
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_SALEOR_QA_WEBHOOK_URL }}
+          JOB_TITLE: "A lot of tests failed, something is probably broken - ${{ needs.run-tests-in-parallel.outputs.dashboard_url }}"
```

Comment:
Maybe we could add here also ENV_URI

Reply:
And link to the dashboard itself where it occurred.

Reply:
In title we will have cypress dashboard url - needs.run-tests-in-parallel.outputs.dashboard_url, in text there will be something like "GithubBot: Tests finished for HTTPS://AUTOMATION-DASHBOARD.STAGING.SALEOR.CLOUD/, result: failure"